packages/components/src/Chips/InternalChipDismissible/InternalChipDismissible.tsx

```diff
@@ -3,33 +3,48 @@ import styles from "./InternalChipDismissible.css";
 import { useInternalChipDismissible } from "./hooks";
 import { InternalChipDismissibleInput } from "./InternalChipDismissibleInput";
 import { InternalChipDismissibleProps } from "./InternalChipDismissibleTypes";
-import { ChipDismissible } from "../ChipDismissible";
+import { InternalChip } from "../InternalChip";
+import { InternalChipButton } from "../InternalChipButton";
 
 export function InternalChipDismissible(props: InternalChipDismissibleProps) {
   const {
     availableChipOptions,
     ref: wrapperRef,
     sortedVisibleChipOptions,
-    handleChipClick,
     handleChipAdd,
+    handleChipClick,
+    handleChipKeyDown,
     handleChipRemove,
     handleCustomAdd,
     handleEmptyBackspace,
+    handleWrapperKeyDown,
   } = useInternalChipDismissible(props);
 
   return (
     <div
       ref={wrapperRef}
       className={styles.wrapper}
       data-testid="dismissible-chips"
+      onKeyDown={handleWrapperKeyDown}
     >
       {sortedVisibleChipOptions.map(chip => (
-        <ChipDismissible
+        <InternalChip
           key={chip.value}
           {...chip}
+          onKeyDown={handleChipKeyDown(chip.value)}
           onClick={handleChipClick(chip.value)}
-          onRequestRemove={handleChipRemove(chip)}
-        />
+          suffix={
+            <InternalChipButton
+              icon="remove"
+              {...chip}
```

Comment:
I don't think we fully need to do a spread, since there is just label, invalid, and disabled that aren't hard-coded, and the rest of the props passed in don't apply.  

Do you think spreading is still okay?  I would say it's okay, since there are enough props there to warrant it, but up to you!

Reply:
the goal is to make it predictable, spreading chip there makes it unpredictable when it starts supporting the same prop name but with a different implementation/expectation